nms/app/packages/magmalte/app/state/AlarmsApiUtil.js

```diff
@@ -39,11 +39,30 @@ export const MagmaAlarmsApiUtil: ApiUtil = {
     console.warn('not implemented');
     return [];
   },
-  getTroubleshootingLink: async ({alertName: _}) =>
-    Promise.resolve({
-      link: '',
+  getTroubleshootingLink: async ({alertName}) => {
+    const alertLinks = [
+      'S1 Setup Failure',
+      'UE attach Failure',
+      'Gateway Checkin Failure',
+      'eNodeB Failure To Connect',
+      'No Metrics Available',
+      'S6A Authentication Success Rate',
+      'Service Request Success Rate',
+      'AGW Reboot',
+      'Expected Service Restart',
+      'Unexpected Service Restart',
+    ];
+    const alert = alertName.toLowerCase();
+    const link = alertLinks.includes(alertName)
+      ? `https://docs.magmacore.org/docs/nms/alerts_troubleshooting#${alert
```

Comment:
let's not hardcode the URL here.

Reply:
@HannaFar I think a good place to put the mapping between alert names and links would be in [this](https://github.com/magma/magma/tree/master/nms/app/packages/magmalte/alerts) directory, where the predefined alerts are specified. It's likely that the predefined alerts and troubleshooting links would change together. cc: @karthiksubraveti

Reply:
Actually add it in https://github.com/magma/magma/blob/master/nms/app/packages/magmalte/data/ directory. This way we can update just the json.